README.md

```diff
@@ -16,33 +16,44 @@ gem 'acts_as_scrubbable'
 
 ## Usage
 
-Simple add the configuration for your fields that map directly to your columns
+Add the configuration for your fields that map directly to your columns and a scrub_type
+for those columns.
 
+Default Scrub types include:
+- `scrub` - scrub the field's value based on it's name or mapping (see mapping case below)
+- `skip` - do not scrub the field's value
+- `wipe` - set the field's value to nil on scrub
+- `sterilize` - delete all records for this model on scrub
 
 ```ruby
-class User < ActiveRecord::Base
+class ScrubExample < ActiveRecord::Base
   ...
 
-  acts_as_scrubbable :first_name, :last_name
+  acts_as_scrubbable :scrub, :first_name # first_name will be random after `scrub!`
+  acts_as_scrubbable :skip, :middle_name # middle_name will be original value after `scrub!`
+  acts_as_scrubbable :wipe, :last_name # last_name will be `nil` after `scrub!`
```

Comment:
i don't love the DSL calling the same method 3 times, but otherwise the concept is sound. is there an easy way to come up with a way to create one dataset for the complete class configuration? maybe something like:
```ruby
acts_as_scrubbable scrub: [:first_name, lat: :latitude],
                   skip: [:middle_name],
                   wipe: [:last_name]
```

Reply:
also, this technique would need to be paired with some validation that when any fields are configured to be scrubbed, skipped or wiped, that *all* fields are configured, right? otherwise 'skip' and literally skipping are the same... or is that something AnonCat would do?

Reply:
ahh, `anonymizer_check`!

Reply:
on further thought, it'd be really cool to be able to define scrub procs here, too. e.g.:
```ruby
acts_as_scrubbable scrub: [gender: -> { %w(M F).sample }]
```

Reply:
`:wipe` scrub type could as easily be a `:nullify` scrub strategy